pipeline-runner/R/gem2s-6-construct_qc_config.R

```diff
@@ -159,7 +159,15 @@ get_sample_mitochondrial_config <- function(scdata_list.sample, config) {
 
 # threshold for doublet score is the max score given to a singlet (above score => doublets)
 get_dblscore_config <- function(scdata_list, config) {
-  probabilityThreshold <- max(scdata_list$doublet_scores[scdata_list$doublet_class == "singlet"], na.rm = TRUE)
+  # if no singlets, set threshold to 0
+  # this is required to prevent the threshold to be "-Inf" when there are no singlets left causing subset experiment to fail
```

Comment:
less verbose!

The experiment that broke was a subset containing doublets only? interesting.

```suggestion
  # if no singlets set threshold to 0, preventing some experiments to fail
```

Reply:
At the beginning I thought it was like that, but actually I think there are some cases in which scDblFinder produces "NA"  values in the "class" column. I saw that in the experiment that was failing. And I think it makes sense because doublets should be already filtered out in the parent experiment, so we shouldn't have any doublets